src/tests/unit/recipeService/createRecipe.test.ts

```diff
@@ -0,0 +1,41 @@
+import { IRecipe } from '../../../interfaces/recipe';
+import { Recipe } from '../../../models/recipeModel';
+import RecipeService from '../../../services/recipeService';
+
+import { mockBody, mockId, mockImage, mockTitle } from '../../mocks/mockRecipe';
+
+import { setupTests } from '../../setupTests';
+
+const recipeService = new RecipeService();
+
+setupTests('createRecipe', () => {
+  test('save new Recipe in a database', async () => {
+    await recipeService.createRecipe(mockId, mockTitle, mockImage, mockBody);
+    const recipe = <IRecipe>await Recipe.findOne({ title: mockTitle });
+    expect(recipe).toBeDefined();
+    expect(recipe.title).toBe(mockTitle);
+    expect(recipe.body).toBe(mockBody);
+    expect(recipe.ownerId).toBe(mockId);
+    expect(recipe.imageUrl).toBe(mockImage);
+  });
+
+  describe('error thrown when', () => {
+    test('no title provided', async () => {
+      await expect(
+        recipeService.createRecipe(mockId, '', mockImage, mockBody)
+      ).rejects.toThrow();
+    });
+
+    test('no password provided', async () => {
```

Comment:
It seems that there is "no body provided", not "no password", which btw is confusing, because body is the entire object you pass in request to server i. e.
`{
  title: string;
  imageUrl: string;
  body: string;
}`